src/wp-includes/block-supports/typography.php

```diff
@@ -468,12 +469,16 @@ function wp_get_computed_fluid_typography_value( $args = array() ) {
 		return null;
 	}
 
+	// Build CSS rule.
+	// Borrowed from https://websemantics.uk/tools/responsive-font-calculator/.
```

Comment:
```suggestion
	/*
	 * Build CSS rule.
	 * Borrowed from https://websemantics.uk/tools/responsive-font-calculator/.
	 */
```

Use multiline comment.